models/yang/common/openconfig-aft-mpls.yang

```diff
@@ -0,0 +1,117 @@
+submodule openconfig-aft-mpls {
```

Comment:
Are we supporting this?

Reply:
I have deleted the file from the branch as it's not needed.